src/reset.css

```diff
@@ -9,3 +9,58 @@ body {
   margin: 0;
   font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
 }
+button {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
```

Comment:
I think this should be `font-family: inherit`. That way, if we change the font in the `html`/`body`, we won't forget to update the button.